package.json

```diff
@@ -43,5 +43,8 @@
         },
         "./*": "./*",
         "./": "./"
+    },
+    "scripts": {
+        "test": "node ./test/runTests.js && node test/validateModuleExportsMatchCommonJS/index.js"
```

Comment:
Previously the publish skipped the `runTests.js` part, but I don't see the harm in including it - it's fast.